test/metabase/cmd/load_and_dump_test.clj

```diff
@@ -0,0 +1,38 @@
+(ns metabase.cmd.load-and-dump-test
+  (:require [clojure.java.io :as io]
+            [clojure.test :refer :all]
+            [metabase.cmd.compare-h2-dbs :as compare-h2-dbs]
+            [metabase.cmd.copy.h2 :as h2]
+            [metabase.cmd.dump-to-h2 :as dump-to-h2]
+            [metabase.cmd.load-from-h2 :as load-from-h2]
+            [metabase.cmd.migrate :as migrate]
+            [metabase.db.connection :as mdb.connection]
+            [metabase.driver :as driver]
+            [metabase.driver.sql-jdbc.connection :as sql-jdbc.conn]
+            [metabase.models.user :refer [User]]
+            [metabase.test :as mt]
+            [metabase.test.data.interface :as tx]
+            [metabase.test.util :as tu]
+            [toucan.db :as db]))
+
+(defn- abs-path
+  [path]
+  (.getAbsolutePath (io/file path)))
+
+(deftest load-and-dump-test
+  (testing "loading data to h2 and porting to DB and migrating back to H2"
+    (let [h2-fixture-db-file (abs-path "frontend/test/__runner__/test_db_fixture.db")
+          h2-file (abs-path "frontend/test/__runner__/test_db_fixture2.db")
```

Comment:
for the destination file I'd just put it in `/tmp` so if you run this locally you don't end up accidentally committing it

Reply:
ah, great. it was my first approach but then thought that maybe in circle-ci the `/tmp/...` paths would collide if different jobs run in parallel.  But it seems it's fine, so I'll change it.